doc/user/assets/sass/_content.scss

```diff
@@ -0,0 +1,193 @@
+a {
+  text-decoration: none;
+
+  &:hover {
+    opacity: 0.75;
+  }
+}
+
+.content {
+  h1, h2, h3, h4, h5, h6 {
+    font-weight: 500;
+
+    a {
+      color: $black;
+      font-weight: 500;
+    }
+
+    + ul {
+      padding-left: 20px;
+    }
+  }
+
+  h1 {
+    font-weight: bold;
+    margin: 16px 0;
+
+    + h2 {
+      margin-top: 0;
+    }
+  }
+
+  h2 {
+    font-size: 22px;
+    margin: 32px 0 8px;
+
+    + h3 {
+      margin-top: 16px;
+    }
+  }
+
+  h3 {
+    font-size: 18px;
+    margin: 28px 0 4px;
+
+    + h4 {
+      margin-top: 8px;
+    }
+  }
+
+  h4 {
+    color: $off-black;
+    font-size: 16px;
+    margin: 28px 0 4px;
+
+    a {
+      color: $off-black;
+    }
+  }
+
+  p, ul > li:first-child > p:first-child {
+    margin: 0;
+  }
+
+  p + p, p + ul, p + ol, p + ol.has-p, ul + p, ul p, ol p, table + p {
+    margin-top: 16px;
+  }
+
+  p + table {
+    margin-top: 8px;
+  }
+
+  p, li {
+    line-height: 1.5;
+  }
+
+  a {
+    color: $purple;
+    font-weight: 500;
+
+    // Coloring of code overrides link coloring; instead insert border-bottom to
+    // indicate hyperlink on code element. This is very prevalent in our docs,
+    // especially among type references.
+    code {
+      border-bottom: 1px dotted $purple-20;
+    }
+  }
+
+  img {
+    max-width: 100%;
+  }
+
+  table {
+    border-collapse: collapse;
+  }
+
+  tr {
+    border-bottom: 1px $grey solid;
+  }
+
+  th {
+    color: $purple;
+    border-bottom: 2px $purple solid;
+    text-align: left;
+    padding: 8px;
+  }
+
+  td {
+    padding: 8px;
```

Comment:
Pages with less spacing between elements are harder to scan; would you mind bumping this up to something larger? I had this at 10 previously, but 12 looks more right with how tight everything else is now.